public/modules/custom/hel_tpm_group/src/Plugin/Field/FieldWidget/GroupSelect2Widget.php

```diff
@@ -35,17 +58,30 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
     $element =  parent::formElement($items, $delta, $element, $form, $form_state); // TODO: Change the autogenerated stub
     $options = $element['#options'];
 
+    $access = $this->fieldPermissionService->getFieldAccess('edit', $items, $this->currentUser, $items->getFieldDefinition());
+
     // If there's more than one option available don't set default value.
```

Comment:
Maybe consider updating the comment to explain why the element access is also changed when there are more than one option.